### ISSUE TITLE
Error when writing IndexHierarchy frame to tsv

### ISSUE THREAD

**Issue by ForeverWintr** [InvestmentSystems/static-frame#114]
```python
f = sf.Frame([[2, 3, 4], [1, 2, 3], [2, 3, 4]], columns=('a', 'b', 'c'))
f.sort_values(('a', 'b')).set_index_hierarchy(('a', 'b')).to_tsv('/tmp/tmp.tsv')
```

Raises: `builtins.TypeError: 'NoneType' object is not subscriptable`

**Comment by ForeverWintr** [comment]
Interestingly, calling repr on the `_index` attribute seems to be a viable workaround:

```python
f = sf.Frame([[2, 3, 4], [1, 2, 3], [2, 3, 4]], columns=('a', 'b', 'c'))
f2 = f.sort_values(('a', 'b')).set_index_hierarchy(('a', 'b'))
repr(f2._index)
f2.to_tsv('/tmp/tmp.tsv')
```

**Comment by flexatone** [comment]
Thanks for posting this issue!

Support for writing and reading IndexHierarchy as index to/from delimited files is supported in 0.5, to be released soon. IndexHierarchy on columns, however, is not yet supported to/from delimited files, but might make into the 0.5 release. I will keep this issue open in regard to columns support.

```python
>>> f = sf.Frame([[2, 3, 4], [1, 2, 3], [3, 3, 4]], columns=('a', 'b', 'c'))

>>> f.sort_values(('a', 'b')).set_index_hierarchy(('a', 'b'), drop=True).to_tsv('/tmp/tmp.tsv')

>>> f.from_tsv('/tmp/tmp.tsv', index_depth=2)
<Frame>
<Index>                  c       <<U1>
<IndexHierarchy>
1                2       3
2                3       4
3                3       4
<int64>          <int64> <int64>
```

**Comment by flexatone** [comment]
Support for reading delimited files with `columns_depth > 1` is now implemented and will be made available in the 0.5 release.  

```python
>>> import static_frame as sf; import numpy as np
>>> f1 = sf.Frame(
...                 np.arange(16).reshape((4,4)),
...                 index=sf.IndexHierarchy.from_product(('I', 'II'), ('a', 'b')),
...                 columns=sf.IndexHierarchy.from_product(('III', 'IV'), (10, 20))
...                 )
>>> f1
<Frame>
<IndexHierarchy>       III     III     IV      IV      <object>
                       10      20      10      20      <object>
<IndexHierarchy>
I                a     0       1       2       3
I                b     4       5       6       7
II               a     8       9       10      11
II               b     12      13      14      15
<<U2>            <<U2> <int64> <int64> <int64> <int64>
>>> f1.to_tsv('/tmp/tmp.txt')

>>> f2 = sf.Frame.from_tsv('/tmp/tmp.txt', index_depth=2, columns_depth=2)
>>> f2
<Frame>
<IndexHierarchy>       III     III     IV      IV      <object>
                       10      20      10      20      <object>
<IndexHierarchy>
I                a     0       1       2       3
I                b     4       5       6       7
II               a     8       9       10      11
II               b     12      13      14      15
<<U2>            <<U2> <int64> <int64> <int64> <int64>
```
